share/cmake/modules/FindIlmBase.cmake

```diff
@@ -56,15 +56,26 @@ if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
 
     # Attempt to find static library first if this is set
     if(ILMBASE_STATIC_LIBRARY)
-        set(_ILMBASE_STATIC 
-            "${CMAKE_STATIC_LIBRARY_PREFIX}Half-${_ILMBASE_LIB_VER}_s${CMAKE_STATIC_LIBRARY_SUFFIX}" 
-            "${CMAKE_STATIC_LIBRARY_PREFIX}Half${CMAKE_STATIC_LIBRARY_SUFFIX}")
+        set(_ILMBASE_STATIC
+            "${CMAKE_STATIC_LIBRARY_PREFIX}Half-${_ILMBASE_LIB_VER}_s${CMAKE_STATIC_LIBRARY_SUFFIX}"
+            "${CMAKE_STATIC_LIBRARY_PREFIX}Half${CMAKE_STATIC_LIBRARY_SUFFIX}"
+        )
+        if(CMAKE_BUILD_TYPE STREQUAL Debug)
+            # WIN32 <= 2.3.0, All platforms > 2.3.0
+            list(INSERT _ILMBASE_STATIC 0
+                "${CMAKE_STATIC_LIBRARY_PREFIX}Half-${_ILMBASE_LIB_VER}_s_d${CMAKE_STATIC_LIBRARY_SUFFIX}")
+        endif()
     endif()
 
     # Find library
+    if(CMAKE_BUILD_TYPE STREQUAL Debug)
```

Comment:
Could you explain a little bit these lines? Is that related to some `Azure` limitations?

Reply:
I'll improve this comment. As an explanation, OpenEXR 2.3.0 and below only add the _d suffix when building for Windows. I observed however that CMake in the current master is being overhauled, and this has been changed so that _d is appended to all Debug builds. I'll double check this.

Reply:
I expounded on this in the install portion of the Find module, and removed earlier instances of the comment where it was less necessary (just a possible known lib name).